specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -103,6 +103,39 @@
         }
       }
     },
+    "/providers/Microsoft.ResourceHealth/emergingIssues": {
```

Comment:
These are read-only "resources" that every user should have access to?  I.e. no RBAC involved with this call?

Reply:
Yes.